src/celeritas/em/model/WentzelModel.cc

```diff
@@ -0,0 +1,170 @@
+                           ImportEmParameters const& em_params,
+                           SPConstImported data)
+    : imported_(data,
+                particles,
+                ImportProcessClass::coulomb_scat,
+                ImportModelClass::e_coulomb_scattering,
+                {pdg::electron(), pdg::positron()})
+{
+    CELER_EXPECT(id);
+
+    ScopedMem record_mem("WentzelModel.construct");
+    HostVal<WentzelData> host_data;
+
+    // This is where the data is built and transfered to the device
+    host_data.ids.action = id;
+    host_data.ids.electron = particles.find(pdg::electron());
+    host_data.ids.positron = particles.find(pdg::positron());
+
+    CELER_VALIDATE(host_data.ids,
+                   << "missing IDs (required for " << this->description()
+                   << ")");
+
+    // Electron mass
+    host_data.electron_mass = particles.get(host_data.ids.electron).mass();
+
+    // TODO: Select form factor
+    host_data.form_factor_type = NuclearFormFactorType::Exponential;
+
+    // Pass user-defined screening factor
+    host_data.screening_factor = em_params.screening_factor;
+
+    // Load Mott coefficients
+    build_data(host_data, materials);
+
+    // Construct data on device
+    data_ = CollectionMirror<WentzelData>{std::move(host_data)};
+
+    CELER_ENSURE(this->data_);
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Particle types and energy ranges that this model applies to.
+ */
+auto WentzelModel::applicability() const -> SetApplicability
+{
+    Applicability electron_applic;
+    electron_applic.particle = this->host_ref().ids.electron;
+    electron_applic.lower = detail::coulomb_scattering_limit();
+    electron_applic.upper = detail::high_energy_limit();
+
+    Applicability positron_applic = electron_applic;
+    positron_applic.particle = this->host_ref().ids.positron;
+
+    return {electron_applic, positron_applic};
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Get the microscopic cross sections for the given particle and material.
+ */
+auto WentzelModel::micro_xs(Applicability applic) const -> MicroXsBuilders
+{
+    return imported_.micro_xs(std::move(applic));
+}
+
+//---------------------------------------------------------------------------//
+//!@{
+/*!
+ * Apply the interaction kernel.
+ */
+void WentzelModel::execute(CoreParams const& params, CoreStateHost& state) const
+{
+    auto execute = make_action_track_executor(
+        params.ptr<MemSpace::native>(),
+        state.ptr(),
+        this->action_id(),
+        InteractionApplier{WentzelExecutor{this->host_ref()}});
+    return launch_action(*this, params, state, execute);
+}
+
+//---------------------------------------------------------------------------//
+#if !CELER_USE_DEVICE
+void WentzelModel::execute(CoreParams const&, CoreStateDevice&) const
+{
+    CELER_NOT_CONFIGURED("CUDA OR HIP");
+}
+#endif
+
```

Comment:
Missing doxygen bracket:

```suggestion
//!@}
```